src/App.js

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const [count, setCount] = useState(0);
+
   return (
     <div className="App">
```

Comment:
I am not sure how i feel about this

Reply:
Looks like you didn't add tests

Reply:
Yes I forgot. I will add them right away!!